wemake_python_styleguide/visitors/tokenize/comments.py

```diff
@@ -60,9 +71,18 @@ def visit_comment(self, token: tokenize.TokenInfo) -> None:
             WrongMagicCommentViolation
 
         """
+        self._line_num = token.start[0]
+        self._in_same_block = (
+            self._line_num - self._prev_comment_line_num == 1 and  # is consec
```

Comment:
NIP: does `consec` mean `consecutive`? if so:

```suggestion
            self._line_num - self._prev_comment_line_num == 1 and  # is consecutive
```

You can even make some properties for this and the next one expressions when a new visitor will be created

Reply:
Thanks for the review. Could you elaborate on "make some properties for this and the next one expressions"? Do you mean having a variable like ``is_consecutive = (self._line_num - self._prev_comment_line_num == 1)``?

Reply:
I mean `@property` but having a well-named variable also sounds good :+1: